crates/iceberg/src/avro/mod.rs

```diff
@@ -18,3 +18,4 @@
 //! Avro related codes.
 #[allow(dead_code)]
 mod schema;
+pub use schema::*;
```

Comment:
```suggestion
pub(crate) use schema::*;
```

Avro schema is not intended for external users.